test/.stats-app/stats-config.js

```diff
@@ -12,7 +21,7 @@ const clientGlobs = [
   },
   {
     name: 'Client Pages',
-    globs: ['.next/static/*/pages/**/*', '.next/static/css/**/*'],
+    globs: ['.next/static/!(images)/pages/**/*', '.next/static/css/**/*'],
```

Comment:
Why exclude images here? Maybe the glob should instead be **/**.js

Reply:
Updated this to be `BUILD_ID` instead which prevents the need to match everything except the `image` folder